cp3pt0-deployment/setup_tenant.sh

```diff
@@ -252,20 +288,28 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 EOF
 )
-
+    title "Checking and authorizing NSS to all namespaces in tenant\n"
     for ns in $SERVICES_NS ${TETHERED_NS//,/ }; do
-        debug1 "Creating following Role:\n"
-        debug1 "${role//ns_to_replace/$ns}\n"
-        echo "${role//ns_to_replace/$ns}" | ${OC} apply -f -
-        if [[ $? -ne 0 ]]; then
-        error "Failed to create Role for NSS in $ns"
-        fi
 
-        debug1 "Creating following RoleBinding:\n"
-        debug1 "${rb//ns_to_replace/$ns}\n"
-        echo "${rb//ns_to_replace/$ns}" | ${OC} apply -f -
-        if [[ $? -ne 0 ]]; then
-        error "Failed to create RoleBinding for NSS in $ns"
+        if [[ $(oc get RoleBinding nss-managed-role-from-$OPERATOR_NS -n $ns 2>/dev/null) != "" ]];then
+            info "RoleBinding nss-managed-role-from-$OPERATOR_NS is already existed in $ns, skip creating"
+        else
+            if $LIMITED;then 
+                error "User only has namespace admin, need to setup role and rolebinding in all tenant namespaces before setup topology"
+            fi
```

Comment:
I think the idea behind is more about providing more error information.

If we are NOT able to get the RoleBinding for target Namespace, could we directly create the role, and return error msg below?